doc/antora/modules/ROOT/pages/index.adoc

```diff
@@ -121,7 +121,7 @@ major version upgrade.
 == Getting Started with FreeRADIUS
 
 FreeRADIUS can be installed using the pre-built packages available
-from http://packages.networkradius.com[Network RADIUS,
+from https://networkradius.com/packages/[Network RADIUS,
```

Comment:
Please leave this as `packages.networkradius.com`.  That is the authoritative link.  The other URL may change over time.

Reply:
But https:// is OK, isn't it?

Reply:
Yes.

Reply:
Fixed.